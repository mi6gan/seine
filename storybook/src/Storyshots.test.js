// @flow
import initStoryshots from '@storybook/addon-storyshots';
import { mount } from 'enzyme';

jest.mock('draft-js/lib/generateRandomKey', () => () => 'mock');

initStoryshots({
  renderer: mount,
});
