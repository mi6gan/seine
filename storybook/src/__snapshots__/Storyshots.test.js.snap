// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Storyshots Docs/Content Bar Chart 1`] = `ReactWrapper {}`;

exports[`Storyshots Docs/Content Column Chart 1`] = `ReactWrapper {}`;

exports[`Storyshots Docs/Content Data 1`] = `ReactWrapper {}`;

exports[`Storyshots Docs/Content Data V 03 1`] = `ReactWrapper {}`;

exports[`Storyshots Docs/Content Flex 1`] = `ReactWrapper {}`;

exports[`Storyshots Docs/Content Image 1`] = `ReactWrapper {}`;

exports[`Storyshots Docs/Content Line Chart 1`] = `ReactWrapper {}`;

exports[`Storyshots Docs/Content Nested Flex 1`] = `ReactWrapper {}`;

exports[`Storyshots Docs/Content Page 1`] = `ReactWrapper {}`;

exports[`Storyshots Docs/Content Pie Chart 1`] = `ReactWrapper {}`;

exports[`Storyshots Docs/Content Table 1`] = `ReactWrapper {}`;

exports[`Storyshots Docs/Content Text 1`] = `ReactWrapper {}`;

exports[`Storyshots Docs/Custom Editor Custom Image Design Editor 1`] = `ReactWrapper {}`;

exports[`Storyshots Docs/Editor Editor Of Bar Chart 1`] = `ReactWrapper {}`;

exports[`Storyshots Docs/Editor Editor Of Column Chart 1`] = `ReactWrapper {}`;

exports[`Storyshots Docs/Editor Editor Of Data 1`] = `ReactWrapper {}`;

exports[`Storyshots Docs/Editor Editor Of Data V 0 3 1`] = `ReactWrapper {}`;

exports[`Storyshots Docs/Editor Editor Of Flex 1`] = `ReactWrapper {}`;

exports[`Storyshots Docs/Editor Editor Of Image 1`] = `ReactWrapper {}`;

exports[`Storyshots Docs/Editor Editor Of Line Chart 1`] = `ReactWrapper {}`;

exports[`Storyshots Docs/Editor Editor Of Nested Flex 1`] = `ReactWrapper {}`;

exports[`Storyshots Docs/Editor Editor Of Pie Chart 1`] = `ReactWrapper {}`;

exports[`Storyshots Docs/Editor Editor Of Table 1`] = `ReactWrapper {}`;

exports[`Storyshots Docs/Editor Editor Of Text 1`] = `ReactWrapper {}`;

exports[`Storyshots Docs/Editor Initial Editor 1`] = `ReactWrapper {}`;

exports[`Storyshots Docs/Editor/API Default layout 1`] = `ReactWrapper {}`;

exports[`Storyshots Tutorials/Block Adding new blocks 1`] = `ReactWrapper {}`;
